apps/view/src/layers.ts

```diff
@@ -0,0 +1,146 @@
+import {
+  TYPE_COPPER,
+  TYPE_SOLDERMASK,
+  TYPE_SILKSCREEN,
+  TYPE_SOLDERPASTE,
+  TYPE_DRILL,
+  TYPE_OUTLINE,
+  TYPE_DRAWING,
+  SIDE_TOP,
+  SIDE_BOTTOM,
+  SIDE_INNER,
+  SIDE_ALL,
+} from 'whats-that-gerber'
```

Comment:
TODO: if we can somehow only import the contants (wtg is CommonJS, so no tree-shaking) we could get a small but meaningful size reduction on this chunk